e2e/fixtures.go

```diff
@@ -44,6 +47,20 @@ func LoadFixtures(fixturesDir string) error {
 	}
 	defer db.Close() // nolint: errcheck
 
+	// Mock current time to ensure consistent exposure calculations.
+	_, err = db.Exec(`CREATE SCHEMA mock;`)
+	if err != nil {
+		return err
+	}
+	_, err = db.Exec(fmt.Sprintf(`CREATE OR REPLACE FUNCTION mock.now() RETURNS TIMESTAMP WITH TIME ZONE LANGUAGE SQL AS $$ SELECT '%s'::timestamptz; $$;`, mockNow))
```

Comment:
Should we format this as "multiline"?

Reply:
You are right, I think it looks better. Done in 6f1f0438e206cddfbc458e371221fabe4b959080.